resources/views/platform/components/beaten-games-leaderboard/leaderboard-card-row.blade.php

```diff
@@ -0,0 +1,20 @@
+@props([
+    'isHighlighted' => false,
+    'myUsername' => null,
+    'paginatedRow' => null,
+])
+
+<?php
+$isHighlighted = $isHighlighted || ($paginatedRow->User === $myUsername);
+?>
+
+<div class="rounded bg-embed p-2 {{ $isHighlighted ? 'my-2' : '' }}" @if ($isHighlighted) style="outline: thin solid;" @endif>
+    <div class="flex items-center justify-between">
+        <div class="flex gap-x-3 items-center">
+            <p class="text-sm">#{{ localized_number($paginatedRow->rank_number) }}</p>
```

Comment:
This can attempt to read a rank_number property on null if you go to a page without any users.

Reply:
Just out of curiosity, how are you landing on a page with no users?

Reply:
I've added an existence check in latest.

Reply:
Yep I'm no longer seeing the issue, I was just selecting a System such as Arcadia 2001 which had no awards in my local environment but now it displays the message that there are no rows matching 👍